### DISCUSSION TITLE
Moving Shlink to another server

### DISCUSSION THREAD

**Discussion by scialom** [shlinkio/discussions#48] [Help wanted] [answered]
* Shlink Version: 12/2019
* PHP Version: 7.4.11
* How do you serve Shlink: Self-hosted Self-hosted nginx
* Database engine used: MySQL

#### Summary
Hello @acelaya 
Our site uses Shlink is moving from Digital Ocean to AWS,
I want to move Shlink as well.
Yet I am newbie and can't find any DNS/ip related stuff.
I have all the credentials: name. url, API-KEY.
Can you please guide me?
Sincerely,
Avraham

**Comment by acelaya** [comment]
I assume by `Shlink Version: 12/2019` that you are still using Shlink 1, as Shlink 2.0.0 was released in January 2020.

First, I would strongly recommend you to update to v2, as v1 is no longer maintained.

If you were using v2, you would be able to set-up a new Shlink clean instance, and then import your short URLs and visits from the old one, as explained in [the docs](https://shlink.io/documentation/advanced/import-short-urls/#shlink-instance).

Maybe it is possible to import from a v1 instance, but I honestly haven't tried, and I don't remember at the moment if the endpoints used for the import were compatible in that version. Most probably not.

If updating is not an option, you could try downloading a dist file for the same version you are using in your old server, then exporting the database with a `mysqldump`, and importing it in your new server.

**Reply by scialom** [reply]
Thank you!
1)
I think it is version 1.
In my case it was done in 12/2019 by previous developer built Shlink instance.
Currently, I consider to build another Shlink instance(v1 or v2) on new aws server, 
export short_urls table data from old DB and export to new DB on new server.
2)
Regarding DNS/connecting domain and ip, it is not so clear to me from docs.
Can you please guide me how to reset DNS/domain to new ip?

**Reply by acelaya** [reply]
The DNS configuration is not specific to Shlink, and it depends on your service provider, so I'm afraid the community cannot help there.

**Comment by scialom** [comment, marked as answer]
It is currently on Digital Ocean and devops can't find it, out bad..
Just looked at https://www.linuxbabe.com/ubuntu/install-shlink-url-shortener-ubuntu-20-04-server
and it directs to https://app.shlink.io/ 
directs to 
https://app.shlink.io/server/create
I thought I would be able to build new instance on new aws server and reset old short domain to new ip.
But it was wrong direction, I accept it.

Thank you very much for your help!
